doc/api/readline.markdown

```diff
@@ -13,7 +13,8 @@ program to gracefully exit:
 
     var rl = readline.createInterface({
       input: process.stdin,
-      output: process.stdout
+      output: process.stdout,
+      history: loadHistorySync()
```

Comment:
it's not clear from the example what is the result of `loadHistorySync()`. It's better to use array literal